tests/dialects/test_builtin.py

```diff
@@ -439,7 +422,7 @@ def test_complex_init():
 
 def test_dense_as_tuple():
     floats = DenseArrayBase.from_list(f32, [3.14159, 2.71828])
```

Comment:
So here we loose precision (because we go to 32 bits), and that's why it's like this?
Just checking if I understand

Reply:
That's my understanding.